_posts/blog/2018-09-12/2018-09-12-how-to-get-more-buyers-on-openbazaar-with-categories-and-tags.markdown

```diff
@@ -0,0 +1,145 @@
+
+![OpenBazaar demo store - How to get more buyers on OpenBazaar with categories and tags](OpenBazaar demo store - How To Get More Buyers on OpenBazar with Categories and Tags.png "OpenBazaar demo store - How to get more buyers on OpenBazaar with categories and tags")
+
+It will make it easier for buyers to shop if you classify your listings. Take time to become familiar with the Store View browsing experience. It will aid you later in listing planning, classification, organization and creation.
+
+#### Classify your Listings for Easy Store Browsing
+
+![Category radio buttons on OpenBazaar](Category radio buttons on OpenBazaar.png "Category radio buttons on OpenBazaar")
+
+A little planning and foresight can make listing creation much easier. Determine which **categories** and possible subcategories you'll support in the beginning. 
+
+In our example store, Variety Bazaar, we have a wide variety of listings. We ensure the potential buyer finds what they want with ease with useful categories.
+
+![Add categories to your OpenBazaar listing](Add categories to your OpenBazaar listing.gif "Add categories to your OpenBazaar listing")
+
+Enter clear, concise and unique category names for your listings. For our example store, we sell clothing, antiques, comic books and fidget spinners. We decided to represent subcategories within clothing by applying more than one category. We also sell retro items within separate categories. These items are all given the extra category 'retro'.
+
+It is important not to confuse tags with categories. Categories are for organizing your store and providing a simple store browsing experience. They are only used for filtering on the store view.
+
+#### Browsing using Listing Filters
+
+When potential buyers browse your store, they will see  a few useful filters to aid them. Understanding these filters will help you plan your store well.
+
+#### Store Search
+
+![Store search bar on OpenBazaar](Store search bar on OpenBazaar.png "Store search bar on OpenBazaar") 
+
+Unlike in the search engines, text entered into this **store search** filters your listings based on their _title_, not their _tags_.
+
+#### Shipping
+
+![Shipping to country and free shipping filter on OpenBazaar](Shipping to country and free shipping filter on OpenBazaar.png "Shipping to country and free shipping filter on OpenBazaar") 
+
+The **Ships to** filter limits listings based on countries to which your listings ship.
+
+The **Free Shipping** filter will toggle display of listings based on this option. 
+
+> Note: This filter displays even if no listings offer free shipping. We encourage you to use it where you can! It makes a purchase more compelling for a buyer if they know they won't also have to deal with extra shipping costs once they get to checkout with your item.
```

Comment:
"get to checkout" sounds awkward, maybe "once they decide to purchase your item."

Reply:
resolved to "at checkout."